packages/next/src/generators/next-auth/generator.ts

```diff
@@ -0,0 +1,56 @@
+import { tsMorphTree, formatFilesWithEslint } from '@ensono-stacks/core';
+import {
+    generateFiles,
+    joinPathFragments,
+    Tree,
+    getProjects,
+    logger,
+    formatFiles,
+} from '@nrwl/devkit';
+import { runTasksInSerial } from '@nrwl/workspace/src/utilities/run-tasks-in-serial';
```

Comment:
should we move this to 'core'?

Reply:
No I dont think so, unless we maintain our own version. not all packages currently depend on core.